src/ProjectTemplates/test/GrpcTemplateTest.cs

```diff
@@ -35,7 +35,7 @@ public ITestOutputHelper Output
     }
 
     [ConditionalFact]
-    [SkipOnHelix("Not supported queues", Queues = "Windows.7.Amd64;Windows.7.Amd64.Open;Windows.81.Amd64.Open;All.OSX;" + HelixConstants.Windows10Arm64 + HelixConstants.DebianArm64)]
+    [SkipOnHelix("Not supported queues", Queues = "All.OSX;" + HelixConstants.Windows10Arm64 + HelixConstants.DebianArm64)]
```

Comment:
While we are touching this, we should probably do SkipHelixArm in the csproj instead of the more fragile skipping of the current 2 arm queues. `<SkipHelixArm>true</SkipHelixArm>`

Reply:
This is a single test `class` among many in ProjectTemplates.Tests and we don't have a `[SkipHelixArm]` attribute. Could however add another special case in `SkipOnHelixAttribute` e.g. `All.Arm64` that's decided using `Contains("arm64")`. Is that your preference❔

In any case, what happens if / when #28090 is fixed on Debian 11 but not Windows 10, or visa versa❔ I was going to fork this branch to narrow the "full matrix" to just Debian 11 Arm64 and remove the queue from `[SkipOnHelix]`s soon.

Reply:
/ping @HaoK

Reply:
Any objections @HaoK?

Reply:
Yeah that's fine, All.Arm64 would be better long term, but whatever is easiest is fine for this PR